### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by pi0**
- Rate limit requests using [reqlim](https://github.com/jsless/reqlim)
- Use `MINIMAL` env to prevent messy bars
- Use `CSB` env to tell process is running in code-sandbox
- Update yarn.lock

### CODE REVIEW

File: nuxt.config.js [hunk 1]
```diff
@@ -45,6 +46,13 @@ export default {
     // See https://github.com/nuxt-community/axios-module#options
   },
 
+  /*
+  ** ServerSide middleware
+  */
+  serverMiddleware: [
+    rateLimit({ windowMs: 3000, max: 3 })
```
Comment (manniL) [inline nuxt.config.js]: Please leave a comment here *why* that middleware is included

Comment (manniL) [reply nuxt.config.js]: Maybe a hint that ppl can/should remove that locally :thinking:

Comment (pi0) [reply nuxt.config.js]: Hmm... I don't know. What do you think of a `codesandbox` module inside repo?

Comment (manniL) [reply nuxt.config.js]: Nice idea! :+1: @pi0

Comment (pi0) [reply nuxt.config.js]: Done

**Pull request merged**